test/query.test.js

```diff
@@ -1483,11 +1483,11 @@ describe('query', () => {
     return Promise.all([p1, p2, p3, p4])
   })
 
-  test('contains', () => {
+  test('contains_field', () => {
```

Comment:
I would keep the `Contains` function tests because this is only deprecated not removed.